csi/util.go

```diff
@@ -85,6 +85,16 @@ func getVolumeOptions(volOptions map[string]string) (*longhornclient.Volume, err
 		}
 	}
 
+	// migratable volumes can only be used in RWX access mode
+	if migratable, ok := volOptions["migratable"]; ok {
+		isMigratable, err := strconv.ParseBool(migratable)
+		if err != nil {
+			return nil, errors.Wrap(err, "Invalid parameter migratable")
+		}
+
+		vol.Migratable = isMigratable && vol.AccessMode == string(types.AccessModeReadWriteMany)
```

Comment:
Should error out if `isMigratable && vol.AccessMode != string(types.AccessModeReadWriteMany)`

Reply:
done